packages/builder/src/components/common/RoleSelect.svelte

```diff
@@ -134,9 +141,16 @@
     getOptionValue={role => role._id}
     getOptionColour={getColor}
     getOptionIcon={getIcon}
-    isOptionEnabled={option =>
-      option._id !== Constants.Roles.CREATOR ||
-      $licensing.perAppBuildersEnabled}
+    isOptionEnabled={option => {
+      if (option._id === Constants.Roles.CREATOR) {
+        return (
+          subType === Constants.PlanType.ENTERPRISE ||
+          (subType === Constants.PlanType.ENTERPRISE_BASIC_TRIAL &&
```

Comment:
This time you need `ENTERPRISE_BASIC` as well? And why is ENTERPRISE_BASIC_TRIAL and `perAppBuildersEnabled` grouped together here?